app/services/generate_manifest_service.rb

```diff
@@ -56,13 +56,13 @@ def default_icon_variant(variant)
   def icon_variant(variant)
     case variant
     when :sm
-      icon.variant(combine_options: combine_options('100x100>'))
+      icon.variant(combine_options('100x100>'))
     when :md
-      icon.variant(combine_options: combine_options('144x144>'))
+      icon.variant(combine_options('144x144>'))
     when :lg
-      icon.variant(combine_options: combine_options('192x192>'))
+      icon.variant(combine_options('192x192>'))
     when :xl3
-      icon.variant(combine_options: combine_options('512x512>'))
+      icon.variant(combine_options('512x512>'))
```

Comment:
Now that the keyword argument `combine_options:` has been removed, I think the method that generates these options should also be renamed. Maybe to `variant_options`? 🤔